### PR TITLE
🔗 Link to Python version of Twitter bot tutorial in Golang version was throwing 404 error. Also some variable naming conflicts were producing errors. Fixed now.

### PR DESCRIPTION
**Pull request by fchikwekwe**
Updated link so that it should be redirecting properly.

### CODE REVIEW

File: content/golang/writing-a-twitter-bot-golang.md [hunk 1]
```diff
@@ -32,13 +32,13 @@ This should create a `go.mod` file within our directory and we should be good to
 
 # Connecting to Twitter
 
-<p>Just like with the <a href="/python/creating-python-twitter-bot-using-twitter-api">Python version of this tutorial</a>, you’ll have to create an app in twitter’s <a href=”https://apps.twitter.com/”>app control panel</a>. Once you’ve created a new application, it should present you with all the secret tokens and keys that you need in order to proceed.</p>
+<p>Just like with the <a href="/python/creating-python-twitter-bot-using-twitter-api-python.md">Python version of this tutorial</a>, you’ll have to create an app in twitter’s <a href=”https://apps.twitter.com/”>app control panel</a>. Once you’ve created a new application, it should present you with all the secret tokens and keys that you need in order to proceed.</p>
```
Comment (elliotforbes) [inline content/golang/writing-a-twitter-bot-golang.md]: I had a quick look, it looks like this should be `<a href="/python/creating-twitter-bot-python/">`

Comment (fchikwekwe) [reply content/golang/writing-a-twitter-bot-golang.md]: No worries. If you look at the latest commit, I did update it. It should actually be ``` <a href="/content/python/creating-twitter-bot-python.md">

Comment (elliotforbes) [reply content/golang/writing-a-twitter-bot-golang.md]: hmm, that breaks it locally on my current version, have you changed anything on the config.toml to resolve the `.md` style links? 

I'll merge and tweak to follow the standard convention.

Seriously though, thank you so much for your contribution!

Comment (fchikwekwe) [reply content/golang/writing-a-twitter-bot-golang.md]: You're welcome!

**Pull request merged**